src/components/SwapForm/OutputCurrencyPanel.tsx

```diff
@@ -30,8 +30,8 @@ const OutputCurrencyPanel: React.FC<Props> = ({
     if (showWrap) {
       return parsedAmountIn?.toExact() || ''
     }
-
-    return parsedAmountOut?.toSignificant(6) || ''
+    if (!parsedAmountOut) return ''
+    return Number(parsedAmountOut.toFixed(currencyOut?.decimals ?? 10)).toString()
```

Comment:
Please check if the `parsedAmountOut` has less than 10 decimals. There can be an error

There's a check at line 85
<img width="815" alt="image" src="https://user-images.githubusercontent.com/19758667/226233294-3048c031-54bd-4e58-ba6c-017007835f77.png">

Reply:
fixed